R/memoise_cran_db.R

```diff
@@ -0,0 +1,22 @@
+# taken from https://github.com/ropenscilabs/packagemetrics/blob/master/R/get_cran.R
+memoise_cran_db <- memoise::memoise({
```

Comment:
Super cool. I'll be using this! I always write an annoying amount of boilerplate to handle this type of thing.

Reply:
functional programming roxxx ;)